cmd/utils/cmd.go

```diff
@@ -60,8 +60,8 @@ const (
 // is redirected to a different file.
 func Fatalf(format string, args ...interface{}) {
 	w := io.MultiWriter(os.Stdout, os.Stderr)
-	if runtime.GOOS == "windows" {
-		// The SameFile check below doesn't work on Windows.
+	if runtime.GOOS == "windows" || runtime.GOOS == "openbsd" {
+		// The SameFile check below doesn't work on Windows neither OpenBSD.
 		// stdout is unlikely to get redirected though, so just print there.
 		w = os.Stdout
```

Comment:
I think we should print to stderr instead of stdout here. I chose stdout a long time ago, for Windows, but it's a strange choice on OpenBSD. One would expect the error message to appear on stderr.

Reply:
Ahh, right. It expects the output on stdout in the test.